core/src/components/range/range.tsx

```diff
@@ -105,6 +105,12 @@ export class Range implements ComponentInterface {
    */
   @Prop() step = 1;
 
+  /**
+   * If `false`, tick marks are not displayed.
```

Comment:
```suggestion
   * If `true`, tick marks are displayed based on the step value.
```

This is super nitpicky but we try to always start each boolean with `If true` 🙂 It will require a npm build though.